contrib/seccomp/seccomp_default.go

```diff
@@ -467,6 +468,18 @@ func DefaultProfile(sp *specs.Spec) *specs.LinuxSeccomp {
 		Syscalls:      syscalls,
 	}
 
+	// include by kernel version
+	if ok, err := kernelversion.KernelGreaterEqualThan(
```

Comment:
kernelversion.GreatherEqualThan()?

https://go.dev/doc/effective_go#names

> For instance, the buffered reader type in the bufio package is called Reader, not BufReader, because users see it as bufio.Reader, which is a clear, concise name.